### DISCUSSION TITLE
Can't assign other users in my app

### DISCUSSION THREAD

**Discussion by hshifaz** [Budibase/budibase#1706] [Help]
Need help in figuring this out!

Lets consider 2 Internal tables, Users (default) and Projects. In this App, each project will have a Manager and it is a Relationship field linked to Users table.

![cant assign new users 3](https://user-images.githubusercontent.com/14042347/121782574-05870b00-cb5f-11eb-9fd1-5ebc7bdd5424.PNG)

In the Data table Project, when creating or updating a record, I can select the default user (user which was created as part of the installation of Budibase). However, other users I've created can't be assigned. The message shows "Row Saved Successfully", but as you can see on the screenshot, it is not.

![cant assign new users 1](https://user-images.githubusercontent.com/14042347/121782590-1cc5f880-cb5f-11eb-9093-42fe1d990d05.PNG)
![cant assign new users 2](https://user-images.githubusercontent.com/14042347/121782601-26e7f700-cb5f-11eb-96d8-6899fce7b8d8.png)

_**I've blur or cut out some personal info.**_

I've checked the following
1- Whether User is active
2- Gave Admin access also, changed to Public to test.

Could anyone help me figure this out, please.

Thank you in Advance.

**Comment by mjashanks** [comment]
Hi there. I believe that this was fixed: https://github.com/Budibase/budibase/issues/1612

Can you try taking an update:

`budi hosting --update`

If that doesn't work for you, let me know - will have to reopen an issue.

Thanks!

**Reply by hshifaz** [reply]
Hi @mjashanks ,
Thanks for replying.

I did an updated just this Morning, but to be sure did it again just now and it says "image is up to date..."

![cant assign new users 4](https://user-images.githubusercontent.com/14042347/121785557-56066480-cb6f-11eb-9f17-86728708de16.PNG)

I'm still experience the issue. like i mentioned before. It works if I select the main user (user created at the installation stage).

**Reply by mjashanks** [reply]
Thanks for letting me know, we'll have to look into it over the next few days.
